modules/gds_printers/manifests/init.pp

```diff
@@ -0,0 +1,20 @@
+# Configure printers for Aviation House.
+#
+# You will need to override `ldap_username` if your LDAP username differs
+# from your local login username.
```

Comment:
I think it's a local username and different from the GitHub login, based on:

- https://github.com/boxen/boxen/blob/v2.6.0/lib/facter/boxen.rb#L10-L19

Could somebody whose local username is different from their GitHub username run this and paste the output?
```
➜  repo git:(gds_printers) ✗ cd /opt/boxen/repo && bundle exec facter boxen_user github_login
boxen_user => dcarley
github_login => dcarley
```